test/root/scripts/TestFamiliarProxy.ash

```diff
@@ -0,0 +1,40 @@
+familiar f = $familiar[blood-faced volleyball];
```

Comment:
I think this test is fine, though it would definitely be better as a proper Java test since you'd get more context on failures if they happened. To my mind, ash tests are better conceptualised as "end to end" tests rather than unit tests.

Reply:
Test was made because it was trivial.  Pick a proxy, copy the "add" lines from code, find and replace to turn them into ash print statements and the code at least is exercised.  I don't like testing a class implicitly by scripts but I will revisit and see if I can do something more class specific.  Thanks.

Reply:
If you check `test/net/sourceforge/kolmafia/textui/RuntimeLibraryTest.java`, we have some tests (e.g. `myPathCoercesToString`) which essentially evaluate a single ASH expression. I think it might be nice to make a generic reusable `attributeEvaluatesTo` parameterized test (as checking an attribute evaluation is something we do a lot), and have it take a stream of loads of possible arguments, including all these proxy values.